plugins/ads/source/Android/src/com/giderosmobile/android/plugins/ads/AdsStateChangeListener.java

```diff
@@ -4,4 +4,5 @@
 	public void onShow();
 	public void onDestroy();
 	public void onHide();
+	public void onRefresh();
```

Comment:
This new onRefresh method must be implemented by all ads provider. Merging this change right now will break all non modified ads providers. Same remark goes for android